sdk/go/common/resource/property_compatibility.go

```diff
@@ -64,6 +64,9 @@ func ToResourcePropertyValue(v property.Value) PropertyValue {
 
 	switch {
 	case len(v.Dependencies()) > 0 || (v.Secret() && v.IsComputed()):
+		if v.IsComputed() {
```

Comment:
Is this change actually needed? Output marshal/unmarshal differently to computed, might be we can leave this as nil which feels more accurate.

Reply:
Tests say it is not needed. Removed.